homeassistant/components/binary_sensor/modbus.py

```diff
@@ -0,0 +1,46 @@
+"""
+Support for Modbus Coil sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.modbus/
+"""
+import logging
+
+import homeassistant.components.modbus as modbus
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+DEPENDENCIES = ['modbus']
+
```

Comment:
I can add it to this PR if that is preffered,  but it is left out on purpous right now. One step at a time..